### ISSUE TITLE
Error: Cannot find module 'jpeg-exif'

### ISSUE THREAD

**Issue by SebastianRuecker** [jhotmann/node-rename-cli#7]
Hey guys,

i've got a tiny litte problem and hope that you can help me :-)

when i try to rename a existing file then it throws me an error

```
> rname -f <previousfilename> <newfilename>
internal/modules/cjs/loader.js:596
    throw err;
    ^

Error: Cannot find module 'jpeg-exif'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:594:15)
    at Function.Module._load (internal/modules/cjs/loader.js:520:25)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/sebastian/.nvm/versions/node/v10.4.1/lib/node_modules/rename-cli/lib/replacements.js:2:14)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
```

thanks :-)

**Comment by SebastianRuecker** [comment]
any suggestions?

**Comment by SebastianRuecker** [comment]
no problem :-)

**Comment by jhotmann** [comment]
Ok I have tracked down the issue, unfortunately it's in one of the dependencies. The fix is easy, though. I have submitted a pull request for the jpeg-exif module and am putting out a patch for rename-cli to use an older version that doesn't have the issue.

**Comment by jhotmann** [comment]
Fixed in [5.1.2](https://github.com/jhotmann/node-rename-cli/releases/tag/5.1.2)

**Comment by SebastianRuecker** [comment]
thanks :-) it works 👍
